src/ThemedControls.cpp

```
@@ -20,6 +24,246 @@ wxColour Blend(const wxColour& a, const wxColour& b, double t) {
+        }
+      }
+      return;
+    }
+    e.Skip();
+  }
+
+  // wxPopupTransientWindow's own outside-click dismissal depends on mouse
+  // capture (only wired on macOS by the base class) and on focus loss (which
+  // never happens here -- see the comment in the ctor); watching the real
+  // global button state sidesteps both and works the same on every platform.
+  void OnIdle(wxIdleEvent& event) {
```

Clang (14.0.6)

[warning] method 'OnIdle' can be made static

(readability-convert-member-functions-to-static)